plugins/mcreator-localization/lang/texts_it_IT.properties

```diff
@@ -3126,4 +3126,4 @@ notification.plugin_load_failed.msg=Impossibile caricare i seguenti plug-in\:
 notification.common.more_info=Ulteriori informazioni
 notification.update_available.msg=È disponibile una nuova versione di MCreator.<br><br>Versione installata\: {0}<br>Nuova versione\: {1}
 notification.patch_available.msg=È disponibile una versione più recente di {0}.<br><br>Versione installata\: {0}.{1}<br>Nuova versione\: {0}.{2}
-notification.plugin_updates.msg=Sono disponibili alcuni aggiornamenti del plug-in
\ No newline at end of file
+notification.plugin_updates.msg=Sono disponibili alcuni aggiornamenti del plug-in
```

Comment:
Same issue here, there's a change that shouldn't be there. I think you removed the line that's after this one.

Reply:
I can guarantee you that I didn't change that line of translation, in any case how can I solve it?

Reply:
I never said you changed that line. I said you deleted the empty line that was the last line of the file, making this line the new last line of the file, hence the change.

Reply:
How can I solve it?